lib/Time/Duration/Concise.pm

```diff
@@ -207,8 +203,9 @@ sub as_concise_string {
     $precision ||= 10;
     my $time_frames         = $self->_duration_array($precision);
     my @concise_time_frames = map {
-        $_ =~ s/\s+//ig;
```

Comment:
Sorry I don't understand.
Do you mean the following code ?
```
map {
   my $t = s/\s+//rg;
   $t =~  /([-|\+]?\d+[A-Za-z]{1})/ig;
  $1;
} @$time_frames;
```

It still not work for `UnusedVars`